src/components/AppBar.js

```diff
@@ -0,0 +1,30 @@
+import { ArrowBack } from '@mui/icons-material';
+import { AppBar as MuiAppBar, IconButton, Toolbar, Typography } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+
+export function AppBar({ title }) {
+  const navigate = useNavigate();
+
+  return (
+    <MuiAppBar position="static" color="default" elevation={1}>
+      <Toolbar>
+        <IconButton 
+          edge="start" 
+          color="inherit" 
+          aria-label="back"
+          onClick={() => navigate(-1)}
+        >
+          <ArrowBack />
+        </IconButton>
+        <Typography variant="h6" component="h1" style={{ 
```

Comment:
The inline styles applied to center the Typography component could be replaced by a more maintainable approach, such as using MUI's sx prop or CSS classes.